general_configuration/_topics/my_settings.md

```diff
@@ -37,25 +32,18 @@ Use the following procedure to change grid and tile icons:
 ![7110](../images/7110.png)
 
   - Set **Show Infrastructure Provider Quadrants** to `ON` to see the
-    four icons in your provider under menu:Compute\[Infrastructure \>
-    Providers\].
+    four icons in your provider under menu: **Compute** > **Infrastructure** > **Providers**.
```

Comment:
should this be one-lined with the start of the sentence?

Reply:
@chessbyte I think yes.  There is a need to do this throughout the docs since they were converted from ascii_doc to markdown.  I try to keep the number and type of updates down for a given PR to make it easier to review. In this case I have completed some formatting type updates while I was updating the navigation menus.